studyzen/Lectures/Lecture.cs

```diff
@@ -0,0 +1,22 @@
+using StudyZen.Common;
+using StudyZen.Utils;
+
+namespace Studyzen.Lectures;
+
+public sealed class Lecture : BaseEntity
+{
+    public int CourseId { get; }
+    public string Name { get; set; }
+    public string? Content { get; set; }
```

Comment:
i don't think a lecture without content is useful :D we can save it as empty string

Reply:
fixed